src/Tizen.WebView/Tizen.WebView/Context.cs

```diff
@@ -51,6 +52,17 @@ public class Context
         private IntPtr _handle;
         private CookieManager _cookieManager;
 
+        private Interop.ChromiumEwk.DownloadStartCallback _downloadStartCallback;
+        
+        /// <summary>
+        /// The callback function for the start download request.
```

Comment:
The delegate for handling download request.

Reply:
Done